charts/devtron/templates/migrator.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if or (eq .Values.installer.mode "hyperion") (eq .Values.installer.mode "Hyperion") }}
+{{- with .Values.hyperion.migrator }}
+---
+# Migrator some things pending
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: postgresql-migrate-devtron
+  annotations:
+    "helm.sh/resource-policy": keep
+spec:
+  ttlSecondsAfterFinished: 1000
+  template:
+    spec:
+      containers:
+      - name: postgresql-migrate-devtron
+        image: {{ .image }}
+        env:
+        - name: GIT_BRANCH
+          value: {{ .envVars.devtron.GIT_BRANCH }}
+        - name: SCRIPT_LOCATION
+          value: {{ .envVars.devtron.SCRIPT_LOCATION }}
+        - name: GIT_REPO_URL
+          value: {{ .envVars.devtron.GIT_REPO_URL }}
+        - name: DB_TYPE
+          value: postgres
+        - name: DB_USER_NAME
+          value: postgres
+        - name: DB_HOST
+          value: postgresql-postgresql.devtroncd
+        - name: DB_PORT
+          value: "5432"
+        - name: DB_NAME
+          value: {{ .envVars.devtron.DB_NAME }}
+        - name: MIGRATE_TO_VERSION
+          value: "0"
+        - name: GIT_HASH
+          value: {{ .envVars.devtron.GIT_HASH }}
+        envFrom:
+          - secretRef:
+              name: postgresql-migrator
+      restartPolicy: OnFailure
+  backoffLimit: 20
+  activeDeadlineSeconds: 1500
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: postgresql-migrate-casbin
+  annotations:
+    "helm.sh/resource-policy": keep
+spec:
+  template:
+    spec:
+      serviceAccountName: devtron
+      containers:
+      - name: devtron-rollout
```

Comment:
we can use same technique in main casbin migration also

Reply:
Done